resume-buddy/src/main/java/com/google/sps/Match.java

```diff
@@ -154,12 +157,25 @@ public static void match(List<Entity> reviewees, List<Entity> reviewers) {
       matchEntity.setProperty("uuid", id.toString());
       datastore.put(matchEntity);
 
+      // Sending emails to reviewee and reviewer
+      try {
+        Email.sendEmail(
+            revieweeEmail,
+            EmailTemplates.MATCH_SUBJECT_LINE_REVIEWEE,
+            EmailTemplates.MATCH_BODY_REVIEWEE,
+            null);
+        Email.sendEmail(
+            reviewerEmail,
+            EmailTemplates.MATCH_SUBJECT_LINE_REVIEWER,
+            EmailTemplates.MATCH_BODY_REVIEWER,
+            null);
+      } catch (IOException io) {
+      }
```

Comment:
If you catch an IOException without any action, it just slips through the cracks. I would at least log the IOException's error message so you can tell if it happens.

Reply:
Just realized this code does not need try catch, because sendEmail function already has code in try catch.